ui/app/src/Views/AlertList/Updates.elm

```diff
@@ -12,61 +12,65 @@ import Utils.Filter exposing (generateQueryString)
 import Views.GroupBar.Updates as GroupBar
 
 
-update : AlertListMsg -> Model -> Filter -> ( Model, Cmd Types.Msg )
-update msg ({ groupBar, filterBar } as model) filter =
-    case msg of
-        AlertsFetched listOfAlerts ->
-            ( { model
-                | alerts = listOfAlerts
-                , groupBar =
-                    case listOfAlerts of
-                        Success alerts ->
-                            { groupBar
-                                | list =
-                                    List.concatMap .labels alerts
-                                        |> List.map Tuple.first
-                                        |> Set.fromList
-                            }
+update : AlertListMsg -> Model -> Filter -> String -> String -> ( Model, Cmd Types.Msg )
+update msg ({ groupBar, filterBar } as model) filter apiUrl baseUrl =
+    let
+        alertsUrl =
+            baseUrl ++ "/#/alerts"
+    in
+        case msg of
+            AlertsFetched listOfAlerts ->
+                ( { model
+                    | alerts = listOfAlerts
+                    , groupBar =
+                        case listOfAlerts of
+                            Success alerts ->
+                                { groupBar
+                                    | list =
+                                        List.concatMap .labels alerts
+                                            |> List.map Tuple.first
+                                            |> Set.fromList
+                                }
 
-                        _ ->
-                            groupBar
-              }
-            , Cmd.none
-            )
+                            _ ->
+                                groupBar
+                  }
+                , Cmd.none
+                )
 
-        FetchAlerts ->
-            let
-                newGroupBar =
-                    GroupBar.setFields filter groupBar
+            FetchAlerts ->
+                let
+                    newGroupBar =
+                        GroupBar.setFields filter groupBar
 
-                newFilterBar =
-                    FilterBar.setMatchers filter filterBar
-            in
-                ( { model | alerts = Loading, filterBar = newFilterBar, groupBar = newGroupBar, activeId = Nothing }
-                , Api.fetchAlerts filter |> Cmd.map (AlertsFetched >> MsgForAlertList)
-                )
+                    newFilterBar =
+                        FilterBar.setMatchers filter filterBar
+                in
+                    ( { model | alerts = Loading, filterBar = newFilterBar, groupBar = newGroupBar, activeId = Nothing }
+                    , Api.fetchAlerts apiUrl filter |> Cmd.map (AlertsFetched >> MsgForAlertList)
```

Comment:
since we're interacting with the api, we want the apiUrl. that includes the `/api/v1` stuff.